tests/servers/tests.py

```diff
@@ -136,3 +136,36 @@ def test_port_bind(self):
         finally:
             if hasattr(TestCase, 'server_thread'):
                 TestCase.server_thread.terminate()
+
+    def test_specified_port_bind(self):
+        """
+        When the developer specifies a port with LiveServerTestCase.port = xyz,
+        ensure that port xyz is actually bound.
+        """
+        TestCase = type(str("TestCase"), (LiveServerBase,), {})
+
+        # Find an open port.
+        try:
+            s = socket.socket()
+            s.bind(('', 0))
+            TestCase.port = s.getsockname()[1]
+            s.close()
+        except socket.error as e:
+            # There are no open ports. What to do??
+            raise
+
+        try:
+            TestCase.setUpClass()
+        except socket.error as e:
+            if e.errno == errno.EADDRINUSE:
```

Comment:
This currently ignores all socket errors except `EADDRINUSE`. I think it should either ignore just `EADDRINUSE` and reraise all other errors like `test_port_bind` does, or simply reraise all errors. In the latter case you can remove the try/except.